src/pages/CreateSpot/CreateSpotsStep1.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useCallback } from 'react';
+import { NavLink, useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { PageWrapperSpaceBetween } from 'components/PageWrapper';
+import { InfoCard } from 'components/InfoCard/InfoCard';
+import { WideButton } from 'components/Button';
+import { Stepper } from 'components/Stepper';
+import { TopNav } from 'components/TopNav/TopNav';
+import { SpotAndPersonaProps } from 'components/CreateSpotAndPersona/SpotAndPersonaProps';
+import ContentImg from 'assets/ContentImg.svg';
+import TeamImg from 'assets/TeamImg.svg';
+import EyeImg from 'assets/EyeImg.svg';
+import SchemeImg from 'assets/SchemeImg.svg';
+import CustomImg from 'assets/CustomImg.svg';
+
+const InformationUnderText = styled(InfoCard)`
+  && .iTLFNI {
+    padding: 0 8px 16px 12px;
+  }
+  && .dVWmBa {
+    ${props => props.theme.typography.subtitle2};
+  }
+  margin: 40px 0 46px 0;
+`;
+
+const ContactUs = styled(NavLink)`
+  ${props => props.theme.typography.subtitle2};
+  margin-left: 4px;
+  color: ${props => props.theme.colors.main.primary};
+  text-decoration: underline;
+`;
+
+export const CreateSpotsStep1: FC = () => {
+  const history = useHistory();
+
+  const onNextClick = useCallback((): void => {
```

Comment:
I am not sure if wrapping that method in useCallback is worth. Renders of WideButton shouldn't be an issue

Reply:
changed